spec/Component.coffee

```diff
@@ -1249,3 +1249,85 @@ describe 'Component', ->
         for item in sample
           sin2.post new IP 'data', item.bar
           sin1.post new IP 'data', item.foo
+
+    describe 'using buffers', ->
+      it 'should get from buffer using a name', (done) ->
+        c = new component.Component
+          inPorts:
+            in:
+              datatype: 'string'
+          outPorts:
+            out:
+              datatype: 'string'
+          process: (input, output) ->
+            return unless input.has 'in', (ip) -> ip.type is 'data'
+            data = input.buffer.get 'in', (ip) -> ip.type is 'data'
```

Comment:
`data` got me confused here, `buffer` or `buf` is a more clear name for what it is.